src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -161,6 +180,8 @@ export const TradeConfirm = ({ history }: RouterProps) => {
         return
       }
 
+      setExecutedTradeAmountConstants(tradeAmountConstants)
```

Comment:
persist values used for the quote before executing the trade - display these with priority over values from form state which may change on us